api/v1/openapi.yaml

```diff
@@ -2399,6 +2402,28 @@ definitions:
       last-error:
         description: Last error seen while performing desired action
         type: string
+  BPFMapSizes:
```

Comment:
I wonder from an API perspective if this should be "BPF Maps" at the top level, then nested within, objects that represent maps with a name and a size. Just in case we end up wanting to include any other information subsequently. It'd be slightly more generic than just "Map sizes".

Reply:
Agree. I changed this to just `BPFMaps` at the top level already. Will also change the lower level to be more generic and allow for other information than just size in the future.

Reply:
Changed this to an array of `BPFMapProperties` which is filled by the agent. The map names are no longer hard-coded in the API. This allows to add/remove maps without changing the API. Currently `BPFMapProperties` contains map name and size but may be extended in the future for additional use cases. PTAL.